docs/kb/semgrep-ci/scan-monorepo-in-parts.md

```diff
@@ -15,6 +15,10 @@ As such, it can be helpful to scan a monorepo in parts for multiple reasons:
 * To improve scan performance in CI and reduce CI run times
 * To logically split the monorepo to simplify managing findings
 
+:::note
+Historical secrets scanning relies on examining the entire repo commit history and cannot be split up by path as other scan types like Code, Supply Chain, or Secrets. As such, it is recommended to turn off historical secrets when splitting up a monorepo by path.
```

Comment:
"Turn off secrets" is kind of vague. How will the user still have coverage if they've purchased all of our scanners? Can we provide recommendations, like "use a separate CI job for historical secrets scans to target the whole monorepo, and another CI job for your typical scans on a monorepo that has been split up" (or similar wording).

Reply:
I agree in principle, but I suspect Sina's reasoning here was that if a repo is too large to complete a regular secrets scan on the available runner, the historical scan is unlikely to be successful. We could spell that out, though, and say something like, if you want to use historical secrets scanning, you'll need a CI runner with enough resources to scan the entire repository in one job. (Maybe with a warning about display names if they do this in a separate job, echoing the later warning.)

Reply:
Noted! I'll leave it to the author and unblock this PR then :)

Reply:
Likewise, I think it's fine as-is but happy to rereview if Sina wants to add detail here.